spec/features/user_cant_sign_in_spec.rb

```diff
@@ -59,6 +59,13 @@
       expect_authenticate_page
     end
 
+    scenario "user tries to log in with unverified account" do
+      user1 = create_user 'user1'
+      email1 = create_email_address_for(user1, 'user-1@example.com')
+      ContactInfo.where(id: email1.id).update_all(verified: false)
+      complete_login_username_or_email_screen('user1')
```

Comment:
The expectation is probably...

```ruby
expect(page).to have_content(t(:'controllers.unverified_user'))
expect_authentication_page # defined in `feature_helpers.rb`
```

I think the code redirects users to the root path, but I think that redirects to the login page (not 100% sure tho)

And if it doesn't pass, useful commands to call would be

`save_and_open_page`

Or somewhere before the last line of the spec scenario put in a `debugger` statement and then poke around in `page`.